src/Exception/InitialisationException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Lexide\PuzzleDI\Exception;
+
+class InitialisationException extends \Exception
```

Comment:
PHP PER coding standards say this should be written as `class InitialisationException extends \Exception {}`

Reply:
Lexide repos don't yet follow PER-2, they're still on PSR-12

Reply:
What sort of a repo is this?!?

Reply:
Shush you!